google-cloud-bigtable/test/google/cloud/bigtable/read_rows_state_machine_acceptance_test.rb

```diff
@@ -0,0 +1,149 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "test_helper"
+
+def read_rows_acceptance_test_data
```

Comment:
I'm not sure that calling this file  an "acceptance" test is consistent with how the term is used elsewhere in this project. It seems to be a unit test collection based on the fixture data in `acceptance/data/read-rows-acceptance-test.json` (which should also be renamed, and moved into the `test` subdir.) Or am I wrong and do these tests interact with the live service somehow?

Reply:
```sh
$ bundle exec rake acceptance
The google-cloud-bigtable gem does not have acceptance tests.
```

Reply:
This test data is being used in unit tests. We have done this in other gems as well.

This PR is just part of the larger bigtable development effort, so I assume full acceptance tests are coming eventually.

Reply:
OK.

Reply:
Read rows test cases are not interacting with the real system it is acceptance test cases to validate read rows state machine. 
`acceptance/data/read-rows-acceptance-test.json` used to emulate different test scenarios for read rows state machine. 
i.e
-  In read, rows response on of the chunk has size 100 bytes but contains only 50  bytes with commit status false then in next chunk remaining data will come. 
- Row chunk has no commit status
- Got row cell value but row key is missing.

I am going to raise another pull request for real system acceptance test cases.